api/app/signals/apps/users/models.py

```diff
@@ -24,12 +24,16 @@ class Profile(CreatedUpdatedModel):
     )
 
     departments = models.ManyToManyField(
-        to='signals.Department'
+        to='signals.Department',
+        related_name='profiles'
     )
 
     # SIG-2016 Added a note field to the profile
     note = models.TextField(null=True, blank=True)
 
+    notification_on_assigment_to_user = models.BooleanField(_('Stuur mij een e-mail als een melding aan mij is toegewezen'), default=False)
+    notification_on_assigment_to_department = models.BooleanField('Stuur mij een e-mail als een melding aan mijn afdeling is toegewezen', default=False)
+
```

Comment:
Nitpicking `notification_on_user_assignment` and `notification_on_department_assignment` 😊 not a blocker at all.

Reply:
Changed this to @vanbuiten's suggestion